### PR TITLE
fix: focus management

### PR DESCRIPTION
**Pull request by otodockal**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/ng-primitives/ng-primitives/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation changes
- [ ] Other... Please describe:

## Issue

Closes #

## What does this PR implement/fix?

To fully support dialog focus management, we need to allow program origin in NgpFocusVisible.
+ fix accordion styling

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/ng-primitives/dialog/src/dialog-trigger/dialog-trigger.directive.ts [hunk 1]
```diff
@@ -24,6 +25,7 @@ import { NgpDialogTriggerToken } from './dialog-trigger.token';
   standalone: true,
   selector: '[ngpDialogTrigger]',
   exportAs: 'ngpDialogTrigger',
+  hostDirectives: [NgpFocusVisible],
```
Comment (ashley-hunter) [inline packages/ng-primitives/dialog/src/dialog-trigger/dialog-trigger.directive.ts]: Unfortunately I don't believe we can have this here, for example, if the trigger is an `ngpButton` then `NgpFocusVisible` would be applied by both `ngpButton` and `ngpDialogTrigger` which would throw an issue.
I guess the solution would be to use `ngpButton` on the trigger in the example.

**Review by ashley-hunter — CHANGES_REQUESTED** [review]
Change looks good, just had one small suggestion. Thank you!

File: packages/ng-primitives/dialog/src/dialog-trigger/dialog-trigger.directive.ts (hunk 1 above)
Comment (otodockal) [reply packages/ng-primitives/dialog/src/dialog-trigger/dialog-trigger.directive.ts]: You're right. I'll send an update. So, the same problem is with ngpAccordionTrigger AFIAK.

Comment (ashley-hunter) [reply packages/ng-primitives/dialog/src/dialog-trigger/dialog-trigger.directive.ts]: Well spotted, that is a good point, the same change should probably be made for it! Thank you!

Comment (otodockal) [reply packages/ng-primitives/dialog/src/dialog-trigger/dialog-trigger.directive.ts]: fixed.

**Review by ashley-hunter — APPROVED** [review]
Great work! Thank you!

**Pull request merged**